bands/templates/band_admin/group_info.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}{% set admin_area=True %}{% set group_type=band.group_type_label|lower %}
+{% block title %}{{ band.group.name }}{% endblock %}
+{% block content %}
+  {% include "band_admin/" + group_type + "_info.html" %}
```

Comment:
The least shared elements of the different groups are the 'info' or agreement sections. For that reason, I opted to simply make a common template that `includes` the group-specific template at-will. That way, the templates are fully customizable for each group type without requiring multiple back-end functions.